.github/workflows/acceptance.yml

```diff
@@ -100,6 +100,12 @@ jobs:
     with:
       testfilter: cache-service
 
+  server-config:
```

Comment:
Looks like now that the `setServerTimeout(server);` is both used in the main server index and in the main spec index, do we still need a whole workflow for this test? Aren't all the workflows now gonna pick up `SERVER_REQUEST_TIMEOUT_MS=60000` from the `localAceeptancetest.env`? That means they will all have the same amount of timeout. Won't it work inside ratelimiter workflow like how you've done it ealier?

Reply:
I'm just asking because there are already a little too many CIs 😓

Reply:
Yes it does, and it opens more questions.  Yes it means that that other tests are also picking up the `server.timeout` setting.  Upon initial implementation it was a comforting feeling to see the other tests reacting to the new timeout setting, so I knew it worked.  I then bumped up the timeout for the tests to increase the allowable time for the tests to finish.  

"Too many CIs" is a relative term that needs to be quantified.  I am okay with many CIs if they are fast, stable and easy to debug.  Fewer long running CIs can also be a nightmare to maintain, as was experienced in this change.  

What is meant by a whole new workflow?  Do we check in the test framework's `index.spec.ts` for server config tests and ONLY set the timeout and other future server config settings for that workflow?